woocommerce/woocommerce-gutenberg-products-block
# Don't check for validation on pushChange

Possibly fixes https://github.com/woocommerce/woocommerce-blocks/issues/8015

In this PR, we revert the check we had for validation before we push to server, this is because any error, before or after, visible or not, would prevent the form from syncing fields.

@mikejolley and I will follow up with a better solution that validates and pushes valid things only.

### Testing steps

1. In Checkout, turn terms and condition block into a checkbox.
2. Go to Checkout, fill out your address, notice that shipping rates update fine and your request makes it to the server.
3. Check the terms and condition block.
4. Back to the form, you should see the same behavior.

### changelog entry
> Fix a bug in which Checkout won't persist anything unless are fields are valid.
